### PR TITLE
chore(event-buffer): Add event buffer notice to UI

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

~~The buffer mechanism seems to be working just fine. Getting it live for everyone is the final test.~~ The scope has changed - see comment below.

### CODE REVIEW

**Comment by Twixes** [comment]
Considering today's change in ingestion infra, I'll create a PR to roll the event buffer out to everyone in https://github.com/PostHog/posthog-cloud-infra. This now is only a UI change to inform users.

> What is the event buffer? Learn by reading this: https://github.com/PostHog/posthog.com/blob/event-buffer-docs/contents/docs/integrate/ingest-live-data.mdx#event-ingestion-nuances, just like users will do.

Added an info notice to the "Listening for events…" step of onboarding and to the Live Events page. Let me know whether this is fits in at all and if the wording is clear to you.

onboarding
live-events

The notice can be closed with the close button on the right. They're then hidden forever in that particular browser.

File: frontend/src/scenes/events/EventBufferNotice.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        return null
+    }
+
+    return (
+        <AlertMessage type="info" style={{ marginBottom: '1rem' }} onClose={acknowledgeEventBuffer}>
+            Note that some events with a never-before-seen distinct ID are deliberately delayed by{' '}
+            {pluralize(preflight?.buffer_conversion_seconds, 'second')}
+            {extended ? ' – this helps ensure accuracy of insights grouped by unique users' : ''}.{' '}
+            <a href="https://posthog.com/docs/integrate/ingest-live-data/#event-ingestion-nuances">
+                Learn more about event buffering in PostHog Docs.
```
Comment (yakkomajuri) [inline frontend/src/scenes/events/EventBufferNotice.tsx:26]:
```suggestion
                Learn more about event buffering in the PostHog Docs.
```

**Review by yakkomajuri — APPROVED** [review]
lgtm

File: frontend/src/scenes/events/EventBufferNotice.tsx (hunk 1 above)
Comment (Twixes) [reply frontend/src/scenes/events/EventBufferNotice.tsx:26]: Hm, I think it's written without the "the", at least in the app

**Review by clarkus — APPROVED** [review]

**Pull request merged**